src/EditorFeatures/CSharpTest/CompleteStatement/CSharpCompleteStatementCommandHandlerTests.cs

```diff
@@ -1994,6 +1994,24 @@ public string Name
             VerifyNoSpecialSemicolonHandling(code);
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CompleteStatement)]
+        public void PropertyAccessors8()
```

Comment:
Test name doesn't reflect the actual test case. We're in a property initializer not accessor.